client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -442,29 +442,17 @@ Undocumented.prototype._sendRequestWithLocale = function( originalParams, fn ) {
 };
 
 /**
- * Determine whether a domain name can be mapped
- *
- * @param {string} domain - The domain name to check.
- * @param {Function} fn The callback function
- * @api public
- */
-Undocumented.prototype.isDomainMappable = function( domain, fn ) {
```

Comment:
That's on me - I've removed the code for checking mappability in `lib/domains/index` in https://github.com/Automattic/wp-calypso/pull/10967/files#diff-a36618f44b2f6aeae8145056394c517aL73, but forgot to remove the actual API call 😞 So I fixed it here ;)